Dockerfile

```diff
@@ -0,0 +1,71 @@
+FROM registry.access.redhat.com/ubi9/ubi:latest
```

Comment:
This is the full `UBI` image rather than `minimal` or `micro` is that deliberate?

Reply:
changed it to `ubi9/openjdk-17-runtime:1.20`